template_content/{% if use_github_actions %}.github{% endif %}/workflows/{% include pathjoin('includes', 'project_name_kebab.jinja') %}-cd.yaml.jinja

```diff
@@ -45,7 +45,7 @@ jobs:
           git config --global user.email "actions@github.com" && git config --global user.name "github-actions"
 
       - name: Deploy to testnet
-        run: algokit deploy testnet --project-name '{{ project_name }}'
+        run: algokit deploy testnet {% if use_workspace %} --project-name '{{ project_name }}' {% endif %}
```

Comment:
Shouldn't we switch to `algokit project deploy` instead?

Reply:
we explicitly deploy to testnet by default, template isn't meant to encourage/include default mainnet deployments out of the box

Reply:
I think Cosimo is talking about using `algokit project deploy testnet` instead of `algokit deploy testnet`.

Reply:
@cusma @neilcampbell oh i see what you mean  - yeah makes sense in that case

Reply:
@aorumbayev @neilcampbell correct, the suggestion was just related to the deprecated `deploy` command!